### PR TITLE
cscope: new, 15.9

### PR DESCRIPTION
**Pull request by AydenMeng**
Topic Description
-----------------

cscope: new, 15.9

Package(s) Affected
-------------------

None

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [ ] AMD64 `amd64`
- [ ] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

- [x] Architecture-independent `noarch`

**Secondary Architectures**

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

### CODE REVIEW

File: app-devel/cscope/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,6 @@
+PKGNAME=cscope
+PKGSEC=devel
+PKGDEP="ncurses glibc"
+PKGDES="Source code browsing tool based on interactive text screen"
+
+ABSHADOW=0
```
Comment (eatradish) [inline app-devel/cscope/autobuild/defines]: Please add a comment explaining why `ABSHADOW=0`

**Review by eatradish — DISMISSED** [review]

File: app-devel/cscope/autobuild/defines (hunk 1 above)
Comment (AydenMeng) [reply app-devel/cscope/autobuild/defines]: The Makefile generated by configure installs the binary to the system directory by default.
Do I need to add this sentence to the define file? By the way, I have grep the define files of other projects and found no similar comments.

Comment (eatradish) [reply app-devel/cscope/autobuild/defines]: @AydenMeng 

> By the way, I have grep the define files of other projects and found no similar comments.

A lot of this is due to the history of irregularly introduced packages, and this is something that we don't have time to deal with yet

Comment (AydenMeng) [reply app-devel/cscope/autobuild/defines]: For some reasons, I just prepared to add commet in the patch, ~~but I noticed that the patch has been merged~~, I should have prepared it earlier, I am very sorry.

**Review by MingcongBai — APPROVED** [review]
Alrighty, ship it!

**Pull request merged**